### PR TITLE
JIRA - BAEL-2773 (The Java NumberFormatException)

### PR DESCRIPTION
**Pull request by abhijoyful**
adding a generic core-java-exceptions module for number format exception and other java exceptions in future

### CODE REVIEW

File: core-java-modules/core-java-exceptions/pom.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.exception.numberformat</groupId>
+    <artifactId>core-java-exceptions</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>NumberFormatExceptionExamples</name>
```
Comment (KevinGilmore) [inline core-java-modules/core-java-exceptions/pom.xml]:
```suggestion
    <name>core-java-exceptions</name>
```

Comment (KevinGilmore) [reply core-java-modules/core-java-exceptions/pom.xml]: As per guidelines, name is optional, but if provided, must be the same as the artifactId (and directory name).

File: core-java-modules/core-java-exceptions/pom.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (KevinGilmore) [inline core-java-modules/core-java-exceptions/pom.xml]: We will also need to add this module to the modules section in the core-java-modules/pom.xml

Comment (abhijoyful) [reply core-java-modules/core-java-exceptions/pom.xml]: > We will also need to add this module to the modules section in the core-java-modules/pom.xml

Done

**Comment by abhijoyful** [comment]
> fix the name tag, add module to pom

Done

File: core-java-modules/core-java-exceptions/src/test/java/com/baeldung/exception/numberformat/NumberFormatExceptionExamplesTest.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+import java.text.ParseException;
+import java.util.Locale;
+import java.util.logging.Logger;
+
+import org.junit.Test;
+
+/**
+ * A set of examples tested to show cases where NumberFormatException is thrown and not thrown.
+ */
+public class NumberFormatExceptionExamplesTest {
```
Comment (KevinGilmore) [inline core-java-modules/core-java-exceptions/src/test/java/com/baeldung/exception/numberformat/NumberFormatExceptionExamplesTest.java]: The unit test class name must end in UnitTest in order to pass the build. Please rename and try again.

**Pull request merged**